nucypher/blockchain/eth/actors.py

```diff
@@ -656,3 +697,342 @@ def request_evaluation(self, evidence):
+        self.economics = TokenEconomics()
+
+        self.__accounts = list()
+        self.__stakers = dict()
+        self.__transacting_powers = dict()
+
+        self.__get_accounts()
+
+        if sync_now and not offline_mode:
+            self.read_onchain_stakes()  # Stakes
+
+    def __str__(self) -> str:
+        r = f'{self.__class__.__name__}(funding_account={self.funding_account})'
+        return r
+
+    def __repr__(self) -> str:
+        r = f'{self.__class__.__name__}(funding_account={self.funding_account})'
+        return r
+
+    #
+    # Configuration
+    #
+
+    def static_payload(self) -> dict:
+        """Values to read/write from stakeholder JSON configuration files"""
+        payload = dict(provider_uri=self.blockchain.provider_uri,
+                       blockchain=self.blockchain.to_dict(),
+                       funding_account=self.funding_account,
+                       accounts=self.__accounts,
+                       stakers=self.__serialize_stakers())
+        return payload
+
+    @classmethod
+    def from_configuration_file(cls, filepath: str = None, **overrides) -> 'StakeHolder':
+        filepath = filepath or cls.default_filepath()
+        payload = cls._read_configuration_file(filepath=filepath)
+
+        # Sub config
+        blockchain_payload = payload.pop('blockchain')
+        blockchain = BlockchainInterface.from_dict(payload=blockchain_payload)
+        payload.update(dict(blockchain=blockchain))
+
+        payload.update(overrides)
+        instance = cls(filepath=filepath, **payload)
+        return instance
+
+    @validate_checksum_address
+    def attach_transacting_power(self, checksum_address: str, password: str = None) -> None:
+        try:
+            transacting_power = self.__transacting_powers[checksum_address]
+        except KeyError:
+            transacting_power = TransactingPower(blockchain=self.blockchain, account=checksum_address)
+            self.__transacting_powers[checksum_address] = transacting_power
+        transacting_power.activate(password=password)
+
+    def to_configuration_file(self, *args, **kwargs) -> str:
+        filepath = super().to_configuration_file(modifier=self.funding_account, *args, **kwargs)
+        return filepath
+
+    def connect(self, blockchain: BlockchainInterface = None) -> None:
+        """Go Online"""
+        if not self.staking_agent:
+            self.staking_agent = StakingEscrowAgent(blockchain=blockchain)
+        if not self.token_agent:
+            self.token_agent = NucypherTokenAgent(blockchain=blockchain)
+        self.blockchain = self.token_agent.blockchain
+
+    #
+    # Account Utilities
+    #
+
+    @property
+    def accounts(self) -> list:
+        return self.__accounts
+
+    def __get_accounts(self) -> None:
+        accounts = self.blockchain.client.accounts
+        self.__accounts.extend(accounts)
+
+    def set_funding_account(self, address: str):
+        self.__funding_account = address
```

Comment:
these getters and setters are a little...  why have them instead of
`self.funding_account = <whatever>`

Reply:
I want to perform checksum address validation on setter here.